<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>851</width>
      <height>811</height>
     </rect>
    </property>
    <property name="title">
     <string>建模</string>
    </property>
    <widget class="QTextEdit" name="create_coordinates">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>801</width>
       <height>741</height>
      </rect>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>30</y>
       <width>16</width>
       <height>741</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="horizontalScrollBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>770</y>
       <width>811</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>811</height>
     </rect>
    </property>
    <property name="title">
     <string>功能选择</string>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>串口配置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>建立坐标系</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>10</y>
      <width>211</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>串口配置</string>
    </property>
    <widget class="QComboBox" name="baud_select">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>98400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>576000</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="serialport_num">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>串口号</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serialport_select">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="baud_rate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>67</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QLabel" name="stop_digit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>67</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QLabel" name="check_digit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>67</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QLabel" name="data_digit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>67</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="stop_select">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="check_method">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>奇校验</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>偶校验</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="data_select">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="open_serialport">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update_serialport">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新串口</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>340</y>
      <width>211</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>参数</string>
    </property>
    <widget class="QLabel" name="UWB1_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>UWB1</string>
     </property>
    </widget>
    <widget class="QLabel" name="UWB2_O">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>UWB2</string>
     </property>
    </widget>
    <widget class="QLabel" name="UWB3_A">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>UWB3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UWB1_display">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UWB2_display">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="UWB3_display">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="UWB4_other">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>UWB4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UWB4_display">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="magnetometer_x_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mag_x_1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mag_x_1_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="magnetometer_y_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mag_y_1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mag_y_1_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="magnetometer_z_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mag_z_1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mag_z_1_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="magnetometer_x_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mag_x_2</string>
     </property>
    </widget>
    <widget class="QLabel" name="magnetometer_z_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mag_z_2</string>
     </property>
    </widget>
    <widget class="QLabel" name="magnetometer_y_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mag_y_2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="groy_x_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="groy_y_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>390</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mag_y_2_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mag_x_2_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="groy_z_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mag_z_2_display">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="gyroscope_x">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gryo_x</string>
     </property>
    </widget>
    <widget class="QLabel" name="gyroscope_z">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gryo_z</string>
     </property>
    </widget>
    <widget class="QLabel" name="gyroscope_y">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gryo_y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="high_display">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="sea_level_distance">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SLD</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="target_y_display">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="target_y">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tar_y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="target_x_display">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="target_x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tar_x</string>
     </property>
    </widget>
    <widget class="QLabel" name="predict_y">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>pre_y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="predict_y_display">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="predict_x_display">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="predict_x">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>pre_x</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>30</y>
      <width>20</width>
      <height>791</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>30</y>
      <width>20</width>
      <height>791</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>serialport_select</tabstop>
  <tabstop>baud_select</tabstop>
  <tabstop>check_method</tabstop>
  <tabstop>data_select</tabstop>
  <tabstop>stop_select</tabstop>
  <tabstop>update_serialport</tabstop>
  <tabstop>open_serialport</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>create_coordinates</tabstop>
  <tabstop>UWB1_display</tabstop>
  <tabstop>UWB2_display</tabstop>
  <tabstop>UWB3_display</tabstop>
  <tabstop>UWB4_display</tabstop>
  <tabstop>mag_x_1_display</tabstop>
  <tabstop>mag_y_1_display</tabstop>
  <tabstop>mag_z_1_display</tabstop>
  <tabstop>groy_x_display</tabstop>
  <tabstop>groy_y_display</tabstop>
  <tabstop>mag_y_2_display</tabstop>
  <tabstop>mag_x_2_display</tabstop>
  <tabstop>groy_z_display</tabstop>
  <tabstop>mag_z_2_display</tabstop>
  <tabstop>high_display</tabstop>
  <tabstop>target_y_display</tabstop>
  <tabstop>target_x_display</tabstop>
  <tabstop>predict_y_display</tabstop>
  <tabstop>predict_x_display</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
